Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1143,4 +1154,54 @@ def problem(text)
       @problems << text
     end
   end
+
+  class TapAuditor
+    attr_reader :name, :path, :tap_audit_exceptions, :problems
+
+    def initialize(tap, strict:)
+      @name                 = tap.name
+      @path                 = tap.path
+      @tap_audit_exceptions = tap.audit_exceptions
+      @strict               = strict
```

Comment:
I'd remove `strict` here until we have a need for it (it doesn't seem there is one yet)?